### PR TITLE
Updating Incident Response guide

### PR DESCRIPTION
**Pull request by alxhrck**
Incident Response guide and checklist are being updated aligned with the latest version of Login.gov's Incident Response Plan included in our FedRAMP package.

### CODE REVIEW

File: _articles/definition-of-done.md [hunk 1]
```diff
@@ -102,7 +102,3 @@ Key items as part of an acceptance thread:
     - Scrum Master
     - Anyone tagged as a reviewer in the ticket
     - A UX team member (if designs were referenced in the ticket)
-
-Once verified, the reviewer can accept the thread:
-- Move the JIRA ticket to Done
-- Link to the acceptance thread in the JIRA ticket (preferably in the comment for the "move to done" step)
```
Comment (zachmargolis) [inline _articles/definition-of-done.md:108]: this is undoing changes I intentionally added in https://github.com/GSA-TTS/identity-handbook/pull/578

Can we undo changes not related to the incident guides?

File: _articles/logs-schema.md [hunk 2]
```diff
@@ -8,18 +8,18 @@ subcategory: "Data Warehouse"
...
 This is a guide to provide the schema definitions for the four log tables in our Data Warehouse:
 
-- `logs.events`
-- `logs.production`
-- `logs.unextracted_events`
-- `logs.unextracted_production`
+- `events.logs`
+- `production.logs`
+- `unextracted_events.logs`
+- `unextracted_production.logs`
```
Comment (zachmargolis) [inline _articles/logs-schema.md:14]: these changes should be backed out too, they were intentional in https://github.com/GSA-TTS/identity-handbook/pull/577/files

File: _articles/incident-response-guide.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
+---
+title: "Incident Response Guide"
+description: "Security Incident Response Guide"
+layout: article
+category: "Security"
+cSpell: ignore sitrep sitreps ISCP
+redirect_from: /articles/secops-incident-response-guide.html
```
Comment (zachmargolis) [inline _articles/incident-response-guide.md:7]: thank you for adding a redirect from a renamed page!

**Review by zachmargolis — APPROVED** [review]

**Comment by alxhrck** [comment]
Closing. Needed to setup commit signing.

**Pull request closed without merging**